src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -8297,6 +8297,40 @@ public void PathMapParser()
             Assert.Equal(1, parsedArgs.Errors.Count());
             Assert.Equal((int)ErrorCode.ERR_InvalidPathMap, parsedArgs.Errors[0].Code);
         }
+
+        [WorkItem(8470, "https://github.com/dotnet/roslyn/issues/8470")]
+        [Fact]
+        public void CompareSymbolDistinguisherDiagnosticArgumentsNoCompilation()
```

Comment:
To hit the issue with errors generated by binding, we need a command-line test since it is the command-line compiler that uses `Diagnostic.GetHashCode`. But we can test more directly (see `DescriptionNoCompilation` test below). I can remove this test if it seems unnecessary.